src/main/java/net/imglib2/view/Views.java

```diff
@@ -1234,4 +1235,19 @@ public static boolean isZeroMin( final Interval interval )
 		return Views.interval( new TransformView< T >( source, transform.inverse() ), source );
 	}
 
+	/**
+	 * Compose two {@link RandomAccessible} sources into a
+	 * {@link RandomAccessible} of {@link Pair}.
+	 *
+	 * @param sourceA
+	 * @param sourceB
+	 * @return
+	 */
+	public static < A, B > RandomAccessiblePair< A, B > pair(
```

Comment:
I would suggest to make the return type `RandomAccessible<Pair<A, B>>` or even `RandomAccessible<? extends Pair<A, B>>`, if there is no strong need for being _exactly_ `RandomAccessiblePair<A, B>`. I can imagine that we might want to add other variants implementing this later.